.github/workflows/publish-docker-fork-visualizer.yml

```diff
@@ -0,0 +1,72 @@
+# Workflow inspired by publish-docker.yml action
+
+name: Create and publish fork-visualizer Docker image
+
+on:
+  push:
+    branches: [ "*" ]
```

Comment:
```suggestion
    branches: [ "main" ]
```

Reply:
So I originally have this run on all branches because if we are not on the main branch then it will build the dev image, this is nice so we can easily pull down the dev branch. But instead I could just tag the normal image with the branch name.